src/VCR/Event/Event.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
You can't add this. PHP-VCR supports PHP 5.3, 5.4 and 5.5 and `declare(strict_types=1);` isn't supported on those versions

Reply:
My bad, got left over from my IDE template I think. Removed.